src/SFA.DAS.EmployerCommitmentsV2.Web/Views/Shared/_DraftApprenticeship.cshtml

```diff
@@ -13,69 +13,105 @@
         </div>
     </fieldset>
 </div>
-<div class="govuk-form-group">
-    <fieldset class="govuk-fieldset">
-        <legend class="govuk-fieldset__legend govuk-fieldset__legend--s">
-            Apprentice name
-        </legend>
-        <div class="govuk-form-group @Html.AddClassIfPropertyInError(x => x.FirstName, "govuk-form-group--error")">
-            <label asp-for="FirstName" class="govuk-label">First name</label>
-            @Html.ValidationMessageFor(m => m.FirstName, null, new { @class = "govuk-error-message", id = "error-message-" + Html.IdFor(m => m.FirstName) })
-            <input asp-for="FirstName" class="govuk-input" />
-        </div>
-        <div class="govuk-form-group @Html.AddClassIfPropertyInError(x => x.LastName, "govuk-form-group--error")">
-            <label asp-for="LastName" class="govuk-label">Last name</label>
-            @Html.ValidationMessageFor(m => m.LastName, null, new { @class = "govuk-error-message", id = "error-message-" + Html.IdFor(m => m.LastName) })
-            <input asp-for="LastName" class="govuk-input" />
-        </div>
-    </fieldset>
-</div>
-<div class="govuk-form-group  @Html.AddClassIfPropertyInError(x => x.DateOfBirth, "govuk-form-group--error")">
-    <fieldset class="govuk-fieldset" aria-describedby="dob-hint" role="group">
-        <legend class="govuk-fieldset__legend govuk-fieldset__legend--s">
-            Date of birth
-        </legend>
-        <span id="dob-hint" class="govuk-hint">
-            For example, 31 3 1980
-        </span>
-        @Html.ValidationMessageFor(m => m.DateOfBirth, null, new { @class = "govuk-error-message", id = "error-message-" + Html.IdFor(m => m.DateOfBirth) })
-        <div class="govuk-date-input">
-            <div class="govuk-date-input__item">
-                <div class="govuk-form-group">
-                    <label asp-for="BirthDay" class="govuk-label govuk-date-input__label">Day</label>
-                    <input asp-for="BirthDay" min="1" max="31" class="govuk-input govuk-date-input__input govuk-input--width-2" type="number">
-                </div>
+
+@if (Model.IsContinuation)
+{
+    @Html.HiddenFor(x => x.FirstName)
+    @Html.HiddenFor(x => x.LastName)
+    @Html.HiddenFor(x => x.BirthDay)
+    @Html.HiddenFor(x => x.BirthMonth)
+    @Html.HiddenFor(x => x.BirthYear)
+    @Html.HiddenFor(x => x.Uln)
+    @Html.HiddenFor(x => x.CourseCode)
+
```

Comment:
They're included so that the values are posted along with the fields the user _can_ edit. The api validates that these fields have not been changed, so our checks can't be overcome.